db/migrate/20220810083954_create_user.rb

```diff
@@ -0,0 +1,12 @@
+class CreateUser < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.text :bio
+      t.integer :postsCounter
```

Comment:
- [ ] please it would be better to follow the convention of creating columns in the database they should be in snake_case like that: comment_counter ...etc please try to do that